Rework sum-of-multiples documentation (#2249)
exercism/problem-specifications: exercises/sum-of-multiples/introduction.md

# Introduction

You work for a company that makes an online game called Enchanted Expeditions.

The game is an adventure survival game.
Players need to travel through treacherous terrain in search of valuable resources.
As they explore, they collect materials like metals, fibers, and wood.
These can be used to craft tools, survival gear, and weapons.

However, crafting requires a special type of magical energy that is only awarded when a player has completed a level.

> The amount of energy awarded depends on which magical items the player found while exploring that level.

Author: I think that works fine, actually. I'd like a second opinion, though, because I've been through so many iterations of this exercise that I'm worried I've lost the plot. @iHiD @ErikSchierboom thoughts?

> You work for a company that makes an online game called Enchanted Expeditions.
> 
> The game is an adventure survival game.
> Players need to travel through treacherous terrain in search of valuable resources.
> As they explore, they collect materials like metals, fibers, and wood.
> These can be used to craft tools, survival gear, and weapons.
> 
> However, crafting requires a special type of magical energy that is only awarded when a player has completed a level.
> 
> The amount of energy awarded depends on which magical items the player found while exploring that level.

Reviewer: ```suggestion
You work for a company that makes an online, fantasy-survival game.

When a player finishes a level, they are awarded energy points.
The amount of energy awarded depends on which magical items the player found while exploring that level.
```

> The amount of energy awarded depends on which magical items the player found while exploring that level.

Reviewer: I made it 5% less dry than @ee7's and I'm happy with this version - agree shortening slightly is good :)

> The amount of energy awarded depends on which magical items the player found while exploring that level.

Author: Thank you!
